pym/bob/generators/EclipseCdtGenerator.py

```diff
@@ -249,7 +266,10 @@ def generateEclipseProject(package, destination, updateOnly, projectName, exclud
         buildMe = []
         buildMe.append("#!/bin/sh")
         buildMe.append("bob dev $1 " + args + " " + project )
-        buildMe.append("bob project -n eclipseCdt " + project + " -u --destination " + destination + ' --name ' + projectName)
+        projectCmd = "bob project -n eclipseCdt " + project + " -u --destination " + destination + ' --name ' + projectName
```

Comment:
Shouldn't all parameters be quoted (via pipes.quote)? Otherwise the shell script might split arguments at white spaces or expand special characters.